packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -293,7 +293,7 @@ class FlutterPlatform extends PlatformPlugin {
 
   /// The test compiler produces dill files for each test main.
```

Comment:
dill or dll?

Reply:
dill is the serialization format for kernel, a Dart intermediate language used by the tooling